fiasco/collections.py

```diff
@@ -363,11 +365,16 @@ def bound_bound_radiative_loss(self, density, **kwargs) -> u.Unit('erg cm3 s-1')
         return rad_loss
 
     @u.quantity_input
-    def free_free_radiative_loss(self) -> u.Unit('erg cm3 s-1'):
+    def free_free_radiative_loss(self, use_itoh=False) -> u.Unit('erg cm3 s-1'):
         r"""
         Calculate the radiative loss rate from free-free emission (bremsstrahlung)
         integrated over wavelength.
 
+        Parameters
+        ----------
+        use_itoh : `bool`, optional
+            Whether to use the Itoh Gaunt Factors.  Defaults to false.
```

Comment:
Would be better to include a citation to the paper here so it is explicit exactly which Itoh Gaunt factors we mean. Could also link to the appropriate method that gives more information about this calculation.